src/components/avatar/avatar.stories.tsx

```diff
@@ -1,49 +1,49 @@
-import type { StoryObj } from '@storybook/react'
-import { Avatar, ElAvatar } from '.'
+import type { Meta, StoryObj } from '@storybook/react'
+import { Avatar } from '.'
 import { Icon } from '../icon'
 
 export default {
   title: 'Components/Avatar',
   component: Avatar,
-}
+  args: {
+    children: 'AD',
+  },
+} as Meta<typeof Avatar>
 
-export const StyleOnlyUsage = {
-  render: ({}) => <ElAvatar>AD</ElAvatar>,
-}
+type Story = StoryObj<typeof Avatar>
+
+/**
+ * An avatar component to be used typically with a `Card` or `Nav` component.
+ * The default variant renders a circle with children.
+ *
+ * There are multiple variants that can be used by utilizing the `data-` props
+ * such as `size`, `colour`, and `shape` (see the HTML version of each version).
+ * In addition to text, it also accepts an Icon as a child.
+ */
```

Comment:
migrate the desc from mdx to here so mdx just need to use `<Description of={stories} />`